src/cristal.tsx

```diff
@@ -14,6 +14,8 @@ export interface CristalProps {
   isResizable?: boolean;
   isDraggable?: boolean;
   onClose?: () => void;
+  onMax?: () => void;
+  onMin?: () => void;
```

Comment:
I think providing callbacks for this is a good idea, but we should have a implementation for the functionality itself. 

Like, pressing maximize, should make the cristal 100% width/height